### ISSUE TITLE
Fails to export certain files.

### ISSUE THREAD

**Issue by HallowSL** [Pherakki/AllStarRumbleModelTool#1]
Files: https://mega.nz/file/wg5FAKKD#DRTt0ugOWnvejdVqzrSQP-w6yL0RKpkQxyDCI282oBs

Most Objects fail to export, along with a few Digimon.

Also for Fighter Digimon it seems the Normal map gtf doesn't convert correctly. https://mega.nz/file/BphCzToI#v1vRaxRLqyboYBFDQPqtARUk3iiL5U4vcOiboX5EIoI

**Comment by Pherakki** [comment]
Yeah, the tool is only intended to work for chr files for the foreseeable future. I might come back to it and add support for additional files when some of my major projects have been wrapped up, if nobody makes a pull request in the meantime.

Uncompressed GTF images don't export well with the tool - you can use e.g. gtf2dds to convert the GTF files the tool extracts from the files in order to convert those properly (I don't have a download link for gtf2dds unfortunately, but it seems to be a well-known program).
